dts/bindings/fs/zephyr,fstab,littlefs.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Description of pre-defined file systems.
+
+compatible: "zephyr,fstab,littlefs"
```

Comment:
maybe: `zephyr,fstab-littlefs`

Reply:
I'd rather not.  There's precedent in Linux for multiple levels:

```
arch/arm/boot/dts/am335x-base0033.dts:          compatible = "ti,tilcdc,slave";
arch/arm/boot/dts/da850-evm.dts:                compatible = "ti,tilcdc,panel";
arch/arm/boot/dts/keystone-clocks.dtsi:         compatible = "ti,keystone,pll-mux-clock";
arch/arm/boot/dts/keystone-clocks.dtsi:         compatible = "ti,keystone,pll-divider-clock";
arch/arm/boot/dts/keystone-clocks.dtsi:         compatible = "ti,keystone,psc-clock";
```

and this isn't an fstab for littlefs, it's a littlefs entry within a `zephyr,fstab`.